genet/utils/elevation.py

```diff
@@ -13,3 +14,48 @@ def get_elevation_data(img, lat, lon):
     elevation_meters = mt.item()
 
     return elevation_meters
+
+
+def validation_report_for_node_elevation(elev_dict, low_limit=-50, mont_blanc_height=4809):
+    """
+    Generates a validation report for the node elevation dictionary.
+    :param elev_dict: contains node_id as key and elevation in meters as value
+    :param low_limit: values below this param get flagged as possibly wrong; set at -50m (below sea level) by default,
+    can optionally set a different value
+    :param mont_blanc_height: values above this param get flagged as possibly wrong; defaults to 4809m,
```

Comment:
lol, so how did we arrive at mont blanc as a benchmark? how come not mont everest? 😂